en/docs/api-management/manage-api-traffic/configure-subscription-approval.md

```diff
@@ -0,0 +1,22 @@
+# Configure Subscription Approval
```

Comment:
This seems to be almost the same content we have drafted under the Administer section right (The page that will probably include info on the process to review subscription requests and proceed to approve or reject)? 
Any reason to repeat the content here?

Reply:
This page will be instructions specific to subscription approval.IDT we should mention that under Administer/Review  ApprovalRequests same reasons as above